akka-actor/src/main/scala/akka/actor/ActorRef.scala

```diff
@@ -512,6 +515,14 @@ final case class SuppressedDeadLetter(message: DeadLetterSuppression, sender: Ac
   require(recipient ne null, "DeadLetter recipient may not be null")
 }
 
+/**
+ * Envelope that is published on the eventStream wrapped in [[akka.actor.DeadLetter]] for every message that is
+ * dropped due to overfull queues or routers with no routees.
+ */
+final case class Dropped(message: Any, reason: String, recipient: ActorRef) extends AllDeadLetters {
+  override def sender: ActorRef = ActorRef.noSender
```

Comment:
Is there never a sender? Or isn't is interesting?

Reply:
it was introduced in Typed and therefore no sender, but maybe we should add it, I'll take a look

Reply:
added sender to Dropped in 06a49aa

Reply:
Looking good, thanks.